src/server/src/test/java/io/cassandrareaper/acceptance/ReaperCassandraIT.java

```diff
@@ -103,7 +103,7 @@ public void upgradeReaperRunner(Optional<String> version) throws InterruptedExce
   }
 
   private static void createReaperTestJettyRunner(Optional<String> version) throws InterruptedException {
-    ReaperTestJettyRunner runner = new ReaperTestJettyRunner(CASS_CONFIG_FILE, version);
+    ReaperTestJettyRunner runner = new ReaperTestJettyRunner(CASS_CONFIG_FILE, version, Optional.empty());
```

Comment:
Even if this added parameter is needed (i don't think it is) this is the wrong value to be passing in here.

Reply:
Removed since I can use `getContext()`